packages/material-ui/src/NoSsr/NoSsr.test.js

```diff
@@ -41,8 +41,8 @@ describe('<NoSsr />', () => {
 
   describe('prop: fallback', () => {
     it('should render the fallback', () => {
-      const wrapper = shallow(
-        <NoSsr fallback="fallback">
+      const wrapper = render(
+        <NoSsr fallback={<p>fallback</p>}>
```

Comment:
This needs to be a component, it doesn't work if it's a string. I think it's something to do with `render`

Reply:
What happens if this is a string? It should work. Sounds like we introduced a breaking change otherwise.

Reply:
`text()` returns a blank string. The same happens if you revert back to the class component

Reply:
If you wrap `NoSsr` in a div you should be able to. `text()` uses the `textContent` of the DOM node an with a plain string there isn't any DOM node.